core/migrations/0001_initial.py

```
@@ -0,0 +1,600 @@
+# Generated by Django 6.0.1 on 2026-01-17 12:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = []
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)